### DISCUSSION TITLE
How to use fitz to delete tables in PDF? Need help, Thanks.

### DISCUSSION THREAD

**Discussion by xiaobie-lhm** [pymupdf/PyMuPDF-Utilities#115] [Q&A] [answered]

**Comment by JorjMcKie** [comment, marked as answer]
Determine the bbox that wraps the table. Then use a redactions annotation to remove the table's text:
```python
page.add_redact_annot(bbox)
page.apply_redactions()
```
Don't forget to save with an appropriate garbage collection to get rid of the "ghost" objects created by the process.
